tests/unit/Trace/Reporter/ZipkinReporterTest.php

```diff
@@ -77,4 +78,30 @@ public function testFormatsTrace()
             }
         }
     }
+
+    public function testSpanKind()
```

Comment:
in hind sight I think it is best to switch this to use zipkin's v2 endpoint. Reason being is that it is easier to reason with the annotations and the upgrade isn't risky to folks https://github.com/openzipkin/zipkin/releases/tag/release-1.30.3

also, the python tracer is using v2 format..